apps/website-25/src/components/Nav.tsx

```diff
@@ -66,36 +67,47 @@ const ExploreSection: React.FC<{
 
 const NavLinks: React.FC<{
   exploreSectionInline?: boolean;
-  children: React.ReactNode;
   courses: Array<{ title: string; href: string; isNew?: boolean }>;
   exploreExpanded: boolean;
   onToggleExplore: () => void;
   className?: string;
+  isScrolled: boolean;
 }> = ({
-  exploreSectionInline, children, courses, exploreExpanded, onToggleExplore, className,
-}) => (
-  <div className={clsx('nav-links flex gap-9', className)}>
-    <div>
-      <button
-        type="button"
-        onClick={onToggleExplore}
-        className="nav-links__dropdown-button flex items-center gap-2 hover:text-bluedot-normal"
-      >
-        Explore
-        <DropdownIcon expanded={exploreExpanded} />
-      </button>
-      {exploreSectionInline && (
-        <ExploreSection
-          expanded={exploreExpanded}
-          courses={courses}
-          className="nav-links__explore-section"
-          innerClassName="pl-6 pt-6"
-        />
-      )}
+  exploreSectionInline,
+  courses,
+  exploreExpanded,
+  onToggleExplore,
+  className,
+  isScrolled,
+}) => {
+  const hoverClasses = clsx('nav-link-animation', isScrolled && 'nav-link-animation-dark');
```

Comment:
we can use available utility [link-on-dark](https://github.com/bluedotimpact/bluedot/blob/f3818377672d3b9728788475f42bab323523c0d2/apps/website-25/src/globals.css#L58) and move the classes from the child selectors below to explicitly cover the nav links:
```suggestion
  const navLinkClasses = clsx('nav-links__link nav-link-animation', isScrolled && 'link-on-dark nav-link-animation-dark');
```

Reply:
These child selectors:
```tsx
isScrolled && '**:text-white [&_a:hover]:text-bluedot-lighter',
```
were mainly there for the explore section (where `**:text-white` applies to both links and non-links), the nav links themselves don't need the `[&_a:hover]:text-bluedot-lighter` class.

I have refactored it to hopefully make this a bit clearer by putting the `[&_a:hover]:text-bluedot-lighter` (now `[&_a:hover]:link-on-dark`) class on the ExploreSection, which also allows removing this bit:
```css
a.nav-link-animation:hover,
button.nav-link-animation:hover {
    @apply text-current;
}
```